Formula/berkeley-db@5.rb

```diff
@@ -4,6 +4,7 @@ class BerkeleyDbAT5 < Formula
   url "https://download.oracle.com/berkeley-db/db-5.3.28.tar.gz"
   sha256 "e0a992d740709892e81f9d93f06daf305cf73fb81b545afe72478043172c3628"
   license "Sleepycat"
+  revision 1
```

Comment:
Do we definitely need a `revision` here?

Reply:
This adds new symbols to the library which Python uses. If anyone has the previous version, Python will error/crash when you use `dbm.ndbm`.

Reply:
Yes, that seems like a good reason 😅